README.md

```diff
@@ -48,78 +48,17 @@ pod 'JSONCodable', '~> 2.1'
 
 ---
 
-`JSONCodable` is made of two separate protocols `JSONEncodable` and `JSONDecodable`.
-`JSONEncodable` allows your structs and classes to generate `NSDictionary` or `[String: AnyObject]` equivalents for use with `NSJSONSerialization`.
-`JSONDecodable` allows you to generate structs from `NSDictionary` coming in from a network request for example.
+**JSONCodable** is made of two separate protocols `JSONEncodable` and `JSONDecodable`.
 
-We'll use the following models in this example:
-```swift
-struct User {
-    let id: Int
-    let name: String
-    var email: String?
-    var company: Company?
-    var friends: [User] = []
-}
-
-struct Company {
-    let name: String
-    var address: String?
-}
-```
```

Comment:
I thought it was not really necessary in understanding the examples and mostly just felt like more clutter than content. Perhaps I am taking for granted the fact that I am familiar with the context though, so I can certainly add it back in if you think it will be clearer.